### PR TITLE
validator release schedule

### LINKED ISSUE #20422: Document Validator release schedule
Inspired by #20420 , the Validator should also have a release and freeze schedule documented. It can either become a part of the general [Release Schedule](https://github.com/ampproject/amphtml/blob/master/contributing/release-schedule.md) document or it's own under validator/.

### PR DESCRIPTION
**Pull request by honeybadgerdontcare**
Fixes #20422

### CODE REVIEW

**Review by Gregable — CHANGES_REQUESTED** [review]
Is this missing a file? I don't see the linked to file in the repo.

**Comment by honeybadgerdontcare** [comment]
> Is this missing a file? I don't see the linked to file in the repo.

You are correct. It has been added.

File: contributing/validator-release-schedule.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Validator Release Schedule
+
+-   [Release Freezes](#release-freezes)
```
Comment (Gregable) [inline contributing/validator-release-schedule.md:3]: This link seems unnecessary given the short doc. The section is likely visible on the screen already.

**Review by Gregable — APPROVED** [review]

File: contributing/validator-release-schedule.md (hunk 1 above)
Comment (honeybadgerdontcare) [reply contributing/validator-release-schedule.md]: Good point, removed!

**Pull request merged**